src/cljs/rems/application.cljs

```diff
@@ -154,7 +154,9 @@
                              (rf/dispatch [::enter-application-page application-id]))
                          (rf/dispatch [::set-status {:status :failed
                                                      :description description
-                                                     :validation (:validation resp)}])))
+                                                     :validation prev-validation
+                                                     ;; this error is only shown if prev-validation was nil
+                                                     :error {:status-text (pr-str (:errors resp))}}])))
```

Comment:
Why `pr-str` the errors and not localized?

Reply:
Yeah eventually we should have localizations for all the errors that the dynamic command endpoint can return. However for now I think this is better than the previous "Failed: ()" message.

Reply:
Maybe you could add these missing features as separate bullets into the issue on the board so we can continue on this task in later PRs?

Reply:
Fair enough

Reply:
Created #798